### ISSUE TITLE
partial interface AnimationEffect exposure set doesn't match original defintion

### ISSUE THREAD

**Issue by foolip** [WICG/animation-worklet#108]
https://wicg.github.io/animation-worklet/#worklet-group-effect has:
```webidl
[Exposed=AnimationWorklet]
partial interface AnimationEffect {
    // Intended for use inside Animation Worklet scope to drive the effect.
    attribute double localTime;
};
```

And https://drafts.csswg.org/web-animations/#the-animationeffect-interface has:
```webidl
[Exposed=Window]
interface AnimationEffect {
    EffectTiming         getTiming();
    ComputedEffectTiming getComputedTiming();
    void                 updateTiming(optional OptionalEffectTiming timing);
};
```

This combination is not valid because https://heycam.github.io/webidl/#Exposed say: "If [Exposed] appears on a partial interface or partial namespace, then the partial’s own exposure set must be a subset of the exposure set of the partial’s original interface or namespace."

In other words, `[Exposed=AnimationWorklet]` needs to be added to https://drafts.csswg.org/web-animations/, or some change made in this spec.

@lukebjerring, I discovered this thanks to a failing "Partial AnimationEffect interface is exposed to 'AnimationWorklet', the original interface is not." test when reviewing https://github.com/web-platform-tests/wpt/pull/12439.

**Comment by majido** [comment]
Thanks for catching this. Is it possible to run the same test locally so I ensure our final solution works before making the change?

FYI, as an alternative I considered if I can put the exposed on the member definition but even in that case, it still needs to be subset of the interface.

> If [Exposed] appears on an interface or namespace member, then the member's exposure set must be a subset of the exposure set of the interface or namespace it is a member of.

**Comment by foolip** [comment]
I found this when rewriting the test to work. I'll send it in a PR so you can try it.

**Comment by majido** [comment]
So to summarize the potential fixes are:
1. Update web-animation/AnimationEffect to include the AnimationWorklet in its exposed list.
2. Change the interface to not be AnimationEffect.

I prefer to do #1. Will send a request to web-animation spec and see how that goes.

**Comment by foolip** [comment]
Test is in https://github.com/web-platform-tests/wpt/pull/12444.

**Comment by majido** [comment]
This issue was moved to w3c/css-houdini-drafts#816
